.yarn/versions/edd3bc49.yml

```diff
@@ -0,0 +1,30 @@
+releases:
+  "@yarnpkg/builder": prerelease
+  "@yarnpkg/cli": prerelease
+  "@yarnpkg/core": prerelease
+  "@yarnpkg/doctor": prerelease
+  "@yarnpkg/plugin-compat": prerelease
+  "@yarnpkg/plugin-constraints": prerelease
+  "@yarnpkg/plugin-dlx": prerelease
+  "@yarnpkg/plugin-essentials": prerelease
+  "@yarnpkg/plugin-exec": prerelease
+  "@yarnpkg/plugin-file": prerelease
+  "@yarnpkg/plugin-git": prerelease
+  "@yarnpkg/plugin-github": prerelease
+  "@yarnpkg/plugin-http": prerelease
+  "@yarnpkg/plugin-init": prerelease
+  "@yarnpkg/plugin-interactive-tools": prerelease
+  "@yarnpkg/plugin-link": prerelease
+  "@yarnpkg/plugin-node-modules": prerelease
+  "@yarnpkg/plugin-npm": prerelease
+  "@yarnpkg/plugin-npm-cli": prerelease
+  "@yarnpkg/plugin-pack": prerelease
+  "@yarnpkg/plugin-patch": prerelease
+  "@yarnpkg/plugin-pnp": prerelease
+  "@yarnpkg/plugin-stage": prerelease
+  "@yarnpkg/plugin-typescript": prerelease
+  "@yarnpkg/plugin-version": prerelease
+  "@yarnpkg/plugin-workspace-tools": prerelease
+  "@yarnpkg/pnp": prerelease
+  "@yarnpkg/pnpify": prerelease
+  vscode-zipfs: prerelease
```

Comment:
Not a blocker, but in this case you only need to bump `plugin-npm` (for direct consumers) and `cli` (for a new binary to be released). The others don't need to be bumped.